api/opencl_platform_layer.asciidoc

```diff
@@ -84,6 +84,43 @@ Otherwise, it returns one of the following errors:
   * {CL_INVALID_VALUE} if _num_entries_ is equal to zero and _platforms_ is
     not `NULL` or if both _num_platforms_ and _platforms_ are `NULL`.
 --
+
+[open,refpage='clIcdGetFunctionAddressForPlatformKHR',desc='Query API entry points for a platform',type='protos']
+--
+The entry point of a given platform that is ICD 2 compatible can be obtained
+using the following function:
+
+include::{generated}/api/protos/clIcdGetFunctionAddressForPlatformKHR.txt[]
+include::{generated}/api/version-notes/clIcdGetFunctionAddressForPlatformKHR.asciidoc[]
+
+  * _platform_ refers to the platform ID returned by {clIcdGetPlatformIDsKHR}
+  * _func_name_ name of an API entry point
+
+{clIcdGetFunctionAddressForPlatformKHR} returns the address of the API entry
+point named by _func_name_. The pointer returned should be cast to a function
+pointer type matching the API entry point as defined in the specification
+header file.
+
+A return value of `NULL` indicates that the specified function does not exist
+for _platform_, or _platform_ is not a valid platform for the implementation.
+--
+
+[open,refpage='clIcdSetPlatformDispatchDataKHR',desc='Set the dispatch data for a platform',type='protos']
+--
+The `dispatch_data` field of a platform can be set using the following function:
+
+include::{generated}/api/protos/clIcdSetPlatformDispatchDataKHR.txt[]
+include::{generated}/api/version-notes/clIcdSetPlatformDispatchDataKHR.asciidoc[]
+
+  * _platform_ refers to the platform ID returned by {clIcdGetPlatformIDsKHR}
+  * _dispatch_data_ is the value to set the `dispatch_data` field of the
+    structure to
+
+{clIcdSetPlatformDispatchDataKHR} returns {CL_SUCCESS} if the function is
+executed successfully.
+It returns {CL_INVALID_PLATFORM} if _platform_ is not a valid platform.
```

Comment:
What is the behavior if _platform_ is an ICD 1 platform?

Reply:
It should return `CL_INVALID_PLATFORM` but the loader should never call this for an ICD 1 platform, either because the platform did not provide `clIcdSetPlatformDispatchDataKHR`or because the loader didn't find the correct tags in the platform dispatch table. If you think this needs additional we can add `ICD 2`to the mention above.